infra/gcp/ensure-staging-storage.sh

```diff
@@ -80,6 +80,8 @@ RELEASE_STAGING_PROJECTS=(
     releng
 )
 
+RELEASE_OLD_STAGING_CLOUDBUILD_ACCOUNT="648026197307@cloudbuild.gserviceaccount.com"
```

Comment:
Nit: move "OLD" to the front

Reply:
Also, maybe comment this to say that this is the GCB build account for the `'kubernetes-release-test'` project, to line up with the comments in line 195.